assets/tasks/common.json

```diff
@@ -612,32 +614,6 @@
                 "color": "terminal.ansiYellow"
             }
         },
-        {
-            "label": "pre-cleanup-x86",
```

Comment:
I see that you remove it, maybe to change to `pre-cleanup-amd64`, but I cannot find the entry for it in the additions...

Reply:
This task was duplicated, so I just removed this one. https://github.com/andreriesco/vscode-torizon-templates/blob/tie-716/assets/tasks/common.json#L591